### PR TITLE
🔀🌐 [fr_FR] add update game SAUVETAGES CÔTIERS

### PR DESCRIPTION
**Pull request by deoxis9001**
**What kind of update does this PR provide?** *Please check*

- [x] new translations / updated translations / translation fixes
- [x] a new feature
- [ ] a new module
- [ ] a bugfix for an existing feature / module
- [ ] improvement of style or user experience
- [x] other: Update game

**Which language(s) did you add/update translations for?**
*use the xx_XX notation for exact language!*
* fr_FR

**What is included in your update?**
* add new poi
* add new building 
* add new school
* add new vehicles

**Additional notes**
in progress

### CODE REVIEW

**Comment by deoxis9001** [comment]
help me :/
I can't find my error

**Comment by jxn-30** [comment]
Building type `25` needs the `startClassrooms` property.

**Comment by deoxis9001** [comment]
ready to merge

File: src/i18n/fr_FR.ts [hunk 1]
```diff
@@ -724,6 +724,69 @@ Si plusieurs demandes échouent dans un court laps de temps, cela peut être dû
...
             possibleBuildings: [11],
         },
+        46: {
+            caption: 'Buggy',
+            color: '#225CB5',
+            coins: 15,
+            credits: 35_000,
+            minPersonnel: 2,
+            maxPersonnel: 4,
+            possibleBuildings: [23],
```
Comment (Suchty112) [inline src/i18n/fr_FR.ts:734]: Is also available in ID 11 of the Compagnie de CRS

File: src/i18n/fr_FR.ts [hunk 2]
```diff
@@ -724,6 +724,69 @@ Si plusieurs demandes échouent dans un court laps de temps, cela peut être dû
...
+        },
+        47: {
+            caption: 'CSL',
+            color: '#225CB5',
+            coins: 12,
+            credits: 6_000,
+            minPersonnel: 0,
+            maxPersonnel: 0,
+            possibleBuildings: [23],
```
Comment (Suchty112) [inline src/i18n/fr_FR.ts:743]: Is also available in ID 11 of the Compagnie de CRS

**Review by Suchty112 — CHANGES_REQUESTED** [review]
image
Then the extension at ID 11 is still missing
and what about https://github.com/deoxis9001/LSSM-V.4/blob/dev/src/modules/extendedCallWindow/i18n/fr_FR.json#L98
the vehiclesByRequirement? xD

**Comment by deoxis9001** [comment]
> image Then the extension at ID 11 is still missing and what about https://github.com/deoxis9001/LSSM-V.4/blob/dev/src/modules/extendedCallWindow/i18n/fr_FR.json#L98 the vehiclesByRequirement? xD
Do you know how to find the code names of the vehicles?

**Comment by Suchty112** [comment]
Singular
1 Buggy, 1 Canot de Sauvetage Léger, 1 Canot Tous Temps, 1 Dragon, 1 Vedette Côtière de Surveillance Maritime

Plural
2 Buggies, 2 Canots de Sauvetage Légers, 2 Canots Tous Temps, 2 Dragons, 2 Vedettes Côtières de Surveillance Maritime

**Review by Suchty112 — APPROVED** [review]

**Pull request merged**